avm/res/net-app/net-app-account/capacity-pool/main.bicep

```diff
@@ -125,6 +125,7 @@ module capacityPool_volumes 'volume/main.bicep' = [
       volumeType: volume.?volumeType
       securityStyle: volume.?securityStyle
       unixPermissions: volume.?unixPermissions
+      throughputMibps: volume.?throughputMibps
```

Comment:
Wasn't added to the `volumeType` and hence cannot be set. I invite you to give it a shot by adding a test value in the module's `max` test 😉